build.gradle

```diff
@@ -51,7 +51,7 @@ dependencies {
 
     implementation 'com.exactpro.th2:common:3.13.4'
     implementation 'com.exactpro.th2:grpc-common:3.1.2'
-    implementation "com.exactpro.th2:grpc-hand:2.7.10"
+    implementation "com.exactpro.th2:grpc-hand:2.8.0-th2_2175-1079853370-SNAPSHOT"
```

Comment:
The version should be changed before merge to use release version, not a snapshot.